Assets/Scripts/PowerNetwork/Grid.cs

```diff
@@ -35,6 +35,21 @@ public bool IsEmpty
             get { return connections.Count == 0; }
         }
 
+        /// <summary>
+        /// Gets the connections to this grid.
+        /// </summary>
+        public HashSet<IPlugable> Connections
```

Comment:
why do you need to expose this. with this, anyone can modify set from outside
 it's used only in UTs now